tests/baselines/reference/classConstructorAccessibility3.errors.txt

```diff
@@ -0,0 +1,77 @@
+tests/cases/conformance/classes/constructorDeclarations/classConstructorAccessibility3.ts(21,1): error TS2322: Type 'typeof Baz' is not assignable to type 'typeof Foo'.
+  Cannot assign a 'protected' constructor type to a 'public' constructor type.
+tests/cases/conformance/classes/constructorDeclarations/classConstructorAccessibility3.ts(22,1): error TS2322: Type 'typeof Qux' is not assignable to type 'typeof Foo'.
+  Cannot assign a 'private' constructor type to a 'public' constructor type.
+tests/cases/conformance/classes/constructorDeclarations/classConstructorAccessibility3.ts(26,1): error TS2322: Type 'typeof Foo' is not assignable to type 'typeof Baz'.
+  Cannot assign a 'public' constructor type to a 'protected' constructor type.
+tests/cases/conformance/classes/constructorDeclarations/classConstructorAccessibility3.ts(27,1): error TS2322: Type 'typeof Bar' is not assignable to type 'typeof Baz'.
+  Cannot assign a 'public' constructor type to a 'protected' constructor type.

```

Comment:
This should be a one-directional relationship: `public` is assignable to `protected` and `private`, and `protected` is assignable to `private` (but none of the reverse)